languages/ruby/exercises/concept/booleans/.docs/introduction.md

```diff
@@ -0,0 +1,61 @@
+## True, False, and Nil
+
+True and false logical states are represented with `true` and `false` in Ruby. These may either be used or returned as literals on their own, or as a result of logical or comparison methods.
+
+```ruby
+true_variable = true
+false_variable = false
+
+true && false
+# => false is returned as a result of the logical method
+
+1 < 2
+# => true is returned as a result of the comparison method
+```
+
+[Nil][nil-dictionary] is an English word meaning "nothing" or "zero". In Ruby, `nil` is a value used to express the _absence_ of an object.
+
+```ruby
+# I do not have a favorite color
+favorite_color = nil
+```
+
+In other programming languages, `null` or `none` values might play a similar role.
+
+## _Truthy_ and _falsey_
+
+In Ruby, all objects evaluate to a _truthy_ or _falsey_ value when they are encountered in a boolean context (like an `if` expression). All data is considered _truthy_ **except** for `false` and `nil`.
+
+## Control flow
+
+_Truthy_ and _falsey_ values are useful in the context of control flow. Like in procedural languages, Ruby has an _if...else_ construct, but it is often more common to use _if_ as a modifier to control the evaluation of an expression.
+
+```ruby
```

Comment:
```ruby
def truthy
  'any object other than nil or false'
end

def falsey
  not truthy
end
```

Does this explain it in a way that makes sense?  The _falsey_ could also be expressed as:

```ruby
def falsey
  [nil, false][rand(2)]
end
```

And if the latter version of `falsey` exists it follows that we could say:

```ruby
def truthy
  not falsey
end
```

Reply:
Could it even be simplified to:

```ruby
def falsey
  nil || false
end
```

While it may always evaluate to false in this definition, it reads as "falsey returns nil or false" which may be easier to grok at an entry level.

Reply:
Yes, or even reversed it would always return `nil`.  It think it is a good expression.

Reply:
````suggestion
```ruby
def falsey
  nil || false
end

def truthy
  not falsey
end

````